src/main/java/com/example/lures/Lures.java

```diff
@@ -0,0 +1,42 @@
+package com.example.lures;
+
+public class Lures {

```

Comment:
URLのパスは複数形ですがJavaのクラス名は単数形でよいですよ。
```suggestion
public class Lure {

```

Reply:
59538fb80b98e811e5741430363b97ca9dcc8a12
クラス名の変更はまだできておりません。
controllerの修正完了した後にファイル名の修正を行いたいと思います。